docs/asciidoc-recommended-practices.adoc

```diff
@@ -25,12 +25,18 @@ NOTE: I recommend _against_ using the +.asc+ extension. The connection to AsciiD
 
 == Text Wrapping
 
-Don't do it!
+Don't wrap the text at a certain amount of characters.
+The recommended way is to write one sentence per line.
 
-Sentence per line? (Neo4j recommendation)
+//todo refer to Neo4j recommendation
 
 AsciiDoc doesn't treat a wrapped line break as a hard line break.
 
+The reader won't notice how you handle line breaks, if the document is rendered to html/epub/pdf.
```

Comment:
I think this is better written as:

"The line breaks between contiguous lines will not be visible in the rendered document (i.e., as the reader sees it)."